code/src/Components/App.js

```diff
@@ -0,0 +1,59 @@
+import React, { useState, useEffect } from 'react';
+
+import PostInput from './PostInput';
+import PostList from './PostList';
+import { MESSAGE_URL } from '../Urls';
+import Loader from './Loader';
+
+import './Style.css';
+
+export const App = () => {
+	const [messages, setMessages] = useState([]);
+	const [isLoading, setLoading] = useState(true);
+	useEffect(() => {
+		fetchMessages();
+		setInterval(fetchMessages, 5000);
```

Comment:
What is the purpose of this interval?
I understand that you probably want to update the list each 10 seconds to provide user with the newest set of data, but we shouldn't do it like this. There is a technique called Subscription which is being used for this kind of communication between front-end and back-end to detect when BE data was updated, but it's clearly based on something far more complicated than doing fetch request each 10 seconds.
For now let's delete this line of code.
Nonetheless, I appreciate your intuition Therese, it's great that you are aware of the fact that data on BE side can be updated without FE concern :)

Reply:
Thanks Maks, I deleted this line of code now. I will read up a bit on subscription and see if I can understand how it can be used! 👍